core/domain/suggestion_registry.py

```diff
@@ -466,6 +487,34 @@ def accept(self, commit_message):
             self.final_reviewer_id, self.target_id, [self.change],
             commit_message, is_suggestion=True)
 
+    def get_all_html_content_strings(self):
+        """Gets all html content strings used in this suggestion.
+
+        Returns:
+            list(str). The list of html content strings.
+        """
+        html_string_list = []
+        html_string_list.append(self.change.translation_html)
+        html_string_list.append(self.change.content_html)
+        return html_string_list
+
+    def convert_html_in_suggestion(self, conversion_fn):
+        """Checks for HTML fields in the suggestion and converts it according
+        to the conversion function.
+
+        Args:
+            conversion_fn: function. The function to be used for converting the
+                HTML.
+        Returns:
+            suggestion change. ExplorationChange. The converted Suggestion
+            change.
+        """
+        self.change.content_html = (
+            conversion_fn(self.change.content_html))
+        self.change.translation_html = (
+            conversion_fn(self.change.translation_html))
+        return exp_domain.ExplorationChange(self.change.to_dict())
```

Comment:
This is a bit confusing if it's an object property then why it's returning a dict, it should do the conversion internally!

Reply:
done, I've changed it to a class method.

Reply:
Any specific reason for making this class object not the property method?

Reply:
As this method is expected to public, I don't think changing/updating the object's property is a good idea inside this function. So rather than updating the object in the function itself, It's better to convert the dict passed to the function and return it. the Migration function can update the suggestion.
Basically I've tried to do this similar to other `convert_html_in` functions implemented in the other state_domain.py.

Reply:
Actually... I don't quite agree. It is safer to edit the object from functions in this object. You are anyway overwriting the  property in a foreign function which IMO is more unsafe than editing it directly. Also I don't get the idea behind not changing objects properties within a public function. That's the basic premise of making object functions right? Because then you restrict what edits can be performed on the object and provide interfaces (functions) to make those changes.